sdk/devcenter/azure-developer-devcenter/tests/conftest.py

```diff
@@ -4,15 +4,17 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from devtools_testutils import test_proxy, add_body_key_sanitizer, add_oauth_response_sanitizer, remove_batch_sanitizers
+from devtools_testutils import test_proxy, add_body_key_sanitizer, add_oauth_response_sanitizer, remove_batch_sanitizers, add_uri_regex_sanitizer
 import pytest
 
```

Comment:
I think the tests are taking so long in playback because `time.sleep` and `asyncio.sleep` aren't being patched to get bypassed (these are used when waiting between polling operations). If you import `is_live` from `devtools_testutils` and `mock` from `unittest`, try adding the following code to the bottom of this file:
```python
@pytest.fixture(scope="session", autouse=True)
def patch_async_sleep():
    async def immediate_return(_):
        return

    if not is_live():
        with mock.patch("asyncio.sleep", immediate_return):
            yield

    else:
        yield


@pytest.fixture(scope="session", autouse=True)
def patch_sleep():
    def immediate_return(_):
        return

    if not is_live():
        with mock.patch("time.sleep", immediate_return):
            yield

    else:
        yield
```

Reply:
Thanks @mccoyp, this solution works, tested locally. I just committed the changes in this PR. Thanks.